ui/raidboss/data/07-dt/hunts/heritage_found.ts

```diff
@@ -162,7 +162,32 @@ const triggerSet: TriggerSet<Data> = {
     },
     // ****** S-RANK: Atticus the Primogenitor ****** //
   ],
-  timelineReplace: [],
+  timelineReplace: [
+    {
+      'locale': 'de',
+      'replaceSync': {
+        'Heshuala': 'Heshuala',
+        'Urna Variabilis': 'Urna Variabilis',
+        'Atticus the Primogenitor': 'Atticus d(?:er|ie|as) Primogenitor',
```

Comment:
It looks like we usually use `(?:der|die|das)` for the article here, if you wanted to keep with convention.